test/resources/PSR with Login/PSR-Login.bru

```diff
@@ -0,0 +1,45 @@
+meta {
+  name: PSR-Login
+  type: http
+  seq: 2
+}
+
+post {
+  url: {{auth}}/government-gateway/session/login
+  body: json
+  auth: none
+}
+
+headers {
+  Authorization: Bearer BXQ3/Treo4kQCZvVcCqKPn7sOksB9wQwDo28f+J2ns1djM1UYrnoRbtOcgt47BuscniXI8OSDhXn90ppX+CnLO9Ek0vYb0NJVplmXggKqS8OOhJ7w0MM2y5rvhyy6F5CI0LPqneRFViZn51rt5XJIkv/Ajp8u4ZYaxEDyMpHh0z9KwIkeIPK/mMlBESjue4V
```

Comment:
Is it right to have this version controlled?

Reply:
removed